zgrab_schema.py

```diff
@@ -761,6 +761,11 @@
     "raw":Binary(),
 })
 
+golang_crypto_param = SubRecord({
+    "value":Binary(),
+    "length":Integer()
```

Comment:
We should still use the new (sized) types for new types. I think that's just an artifact of when we wrote the ZGrab schema, and it's (relative) unuse in any part of Censys now.